src/javaRosa.js

```diff
@@ -1117,9 +1117,7 @@ define([
 
         var items = getItextItemsFromMugs(form, true);
         items = _.object(_.map(items, function (item) { return [item.id, item]; }));
-        cells = nextRow();
-        while (cells) {
-            // what's the point of creating items here?
+        for (cells = nextRow(); cells; cells = nextRow()) {
```

Comment:
I think it's if it hits https://github.com/dimagi/Vellum/pull/420/files#diff-e5b3eca8407b45b132e0147b8392192cR1124 then it won't increment the row.

g2m when tests are done